docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -91,7 +91,10 @@ used. One of the following approaches can be adopted to fix the unmanaged indice
 . Manually delete the indices when they are no longer needed.
 . Explicitly configure APM data streams with the default data stream lifecycle config.
 Using this approach would migrate all data streams to use data stream lifecycles,
-which should be equivalent to the default ILM policies:
+which should be equivalent to the default ILM policies. Note that if you have a custom
+lifecycle policy in place for any of the datastreams then you don't need to do anything
+for those datastreams as they are not affected by this bug. Going forward, for using
+custom ILM policies with new datastreams, please consult the {ref}/apm-ilm-how-to.html[updated guide].
```

Comment:
Why do you think this belongs here? If I have a custom ILM policy I don't even read this line due to 
>The issue also does not occur if a custom ILM policy is configured using a custom component template.

Reply:
Took a stab at updating this in a different way to avoid confusion. PTAL.

Reply:
This part looks good to me now - are you planning on also adding the `prefer-ilm` section to this PR or a separate one? (The PR description is not updated)